src/SomeStyle.js

```diff
@@ -0,0 +1,26 @@
+// eslint-disable-next-line import/no-anonymous-default-export
+export default () => ({
```

Comment:
what is a better name for this file?

Reply:
as far as i can tell it is not even used.. can we just remove it?